bin/gce-delete-instance

```diff
@@ -1,9 +1,48 @@
 #!/bin/bash
 
-uage="Usage: `basename $0` <instance_name> [-q | --quite]"
+delete-instance() {
+  uage="Usage: `basename $0` <instance_name> [-q | --quite]"
```

Comment:
Fixed in the latest commit. https://github.com/solvcon/solvcon-gce/pull/19/commits/f338780584502392e3ef7e3982244c7e47f2f199